apollo-router/tests/snapshots/integration_tests__traced_basic_composition.snap

```diff
@@ -107,11 +128,16 @@ expression: get_spans()
                                      },\n                                    ),\n                                ],\n                                variable_usages: [],\n                                operation: \"query($representations:[_Any!]!){_entities(representations:$representations){...on Product{name}}}\",\n                            },\n                        ),\n                    },\n                ),\n                Flatten(\n                    FlattenNode {\n                        path: Path(\n                            [\n                                Key(\n                                    \"topProducts\",\n                                ),\n                                Flatten,\n                                Key(\n                                    \"reviews\",\n                                ),\n                                Flatten,\n                                Key(\n                                    \"author\",\n                                ),\n                            ],\n                        ),\n                        node: Fetch(\n                            FetchNode {\n                                service_name: \"accounts\",\n                                requires: [\n                                    InlineFragment(\n                                        InlineFragment {\n                                            type_condition: Some(\n                                                \"User\",\n                                            ),\n                                            selections: [\n                                                Field(\n                                                    Field {\n                                                        alias: None,\n                                                        name: \"__typename\",\n                                                        selections: None,\n                                                    },\n                                                ),\n                                                Field(\n                                                    Field {\n                                                        alias: None,\n                                                        name: \"id\",\n                                                        selections: None,\n                                                    },\n                                                ),\n                                            ],\n                                        },\n                                    ),\n                                ],\n                                variable_usages: [],\n                                operation: \"query($representations:[_Any!]!){_entities(representations:$representations){...on User{name}}}\",\n                            },\n                        ),\n                    },\n                ),\n            ],\n        },\n    ],\n}"
```

Comment:
why is this appearing now? I do not see a change in the query planner code that would mean it is now generated

Reply:
I don't know is the quick answer. It may be related to the fact I have a newer version of insta on my branch than on main. (1.12.0 compared to 1.8.0).

Reply:
that one is generated by @o0Ignition0o's test-span crate IIRC

Reply:
I think this is resolved. Not sure which change resolved it though...